cypress/integration/delegates/delegates.page.spec.ts

```diff
@@ -25,7 +25,7 @@ describe('Delegates Page', () => {
     setAccount(TEST_ACCOUNTS.normal, () => {
       // Find the shadow delegates
       cy.contains('Shadow Delegates').should('be.visible');
-      cy.contains('Constitutional Delegates').should('be.visible');
+      cy.contains('aligned Delegates').should('be.visible');
```

Comment:
This might be looking for capital-A "Aligned Delegates"

Reply:
Thanks!